core/backend/src/IModelDb.ts

```diff
@@ -1144,7 +1144,7 @@ export abstract class IModelDb extends IModel {
           if (ret.error !== undefined)
             reject(new Error(ret.error.message));
           else
-            resolve(ret.result);
+            resolve(ret.result!);

```

Comment:
We could probably avoid the need for this by making `ErrorStatusOrResult` in NativeLibrary.d.ts a union type, but that would need to be a separate PR...